src/main/flight/imu.c

```diff
@@ -323,18 +323,10 @@ STATIC_UNIT_TESTED void imuUpdateEulerAngles(void)
     }
 }
 
-static bool imuIsAccelerometerHealthy(const float *accAverage)
+static bool imuIsAccelerometerHealthy(void)
 {
-    float accMagnitudeSq = 0;
-    for (int axis = 0; axis < 3; axis++) {
-        const float a = accAverage[axis];
-        accMagnitudeSq += a * a;
-    }
-
-    accMagnitudeSq = accMagnitudeSq * sq(acc.dev.acc_1G_rec);
-
     // Accept accel readings only in range 0.9g - 1.1g
-    return (0.81f < accMagnitudeSq) && (accMagnitudeSq < 1.21f);
+    return (-0.9f < acc.accMagnitude) && (acc.accMagnitude < 1.1f);
```

Comment:
The minus sign "survived" on accident
```suggestion
    return (0.9f < acc.accMagnitude) && (acc.accMagnitude < 1.1f);
```

Reply:
Whoops !!!  Thank goodness you're here !